infra/base-images/base-runner/collect_dft

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -u
```

Comment:
This will be deleted when migrating to CF, it's just easier for now to re-use all this existing logic for finding binaries and running them in parallel (copied from the `coverage` script)

Reply:
and on ClusterFuzz it will not need the multiprocessing, as it'll be one target at a time